packages/rocketchat-lib/server/startup/settings.js

```diff
@@ -1444,7 +1444,10 @@ RocketChat.settings.addGroup('Layout', function() {
 			type: 'string',
 			'public': true
 		});
-		this.add('Layout_Home_Body', 'Welcome to Rocket.Chat <br> Go to APP SETTINGS -> Layout to customize this intro.', {
+		this.add('Layout_Home_Body', '<p>Welcome to Rocket.Chat!</p>
+<p>The Rocket.Chat desktops apps for Windows, macOS and Linux are available to download <a title="Rocket.Chat desktop apps" href="https://rocket.chat/download" target="_blank" rel="noopener">here</a>.</p>
```

Comment:
you'll have to keep everything on the same line. this is not a valid javascript string.